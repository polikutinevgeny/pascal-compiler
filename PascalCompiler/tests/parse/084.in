type
    t = Integer;
    g = Real;
begin
end.