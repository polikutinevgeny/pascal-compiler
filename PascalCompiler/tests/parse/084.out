program 
  integer: (t = (built-in));
  real: (g = (built-in));
  char: (char = (built-in));
  t: (t = (built-in));
  g: (g = (built-in));
